src/Features/CSharp/Portable/Completion/CompletionProviders/SymbolCompletionProvider.cs

```diff
@@ -83,5 +85,20 @@ protected override CompletionItemRules GetCompletionItemRules()
         {
             return ItemRules.Instance;
         }
+
+        protected override Task<IEnumerable<ISymbol>> GetPreselectedSymbolsWorker(AbstractSyntaxContext context, int position, OptionSet options, CancellationToken cancellationToken)
+        {
+            var recommender = context.GetLanguageService<IRecommendationService>();
+            var typeInferrer = context.GetLanguageService<ITypeInferenceService>();
+
+            var inferredTypes = typeInferrer.InferTypes(context.SemanticModel, position, cancellationToken)?.ToSet();
+            if (inferredTypes == null || !inferredTypes.Any())
+            {
+                return SpecializedTasks.EmptyEnumerable<ISymbol>();
+            }
+
+            var symbols = recommender.GetRecommendedSymbolsAtPosition(context.Workspace, context.SemanticModel, position, options, cancellationToken);
```

Comment:
Yes. In theory it shouldn't be a problem because it basically just calls LookupSymbols, but maybe we should try to avoid that.